zsh/aliases.zsh

```diff
@@ -0,0 +1,60 @@
+# Aliases with existence checks for portability
+
+# Basic aliases that should work everywhere
```

Comment:
get rid of this comment, it doesn't provide value. For the others accross this PR, we don't need to comment on universality or dependency checks